### PR TITLE
Quote connection attributes

### LINKED ISSUE #15: ODBC Connection attribute values should be quoted
Connection attribute values which include some of these symbols: `[]{}(),;;?*=!@.` must be quoted with curly braces `{}` in the connection string provided to `SQLDriverConnect`.

See https://msdn.microsoft.com/en-us/library/ms715433(v=vs.85).aspx#Anchor_4

### PR DESCRIPTION
**Pull request by jgoizueta**
Fixes #15

### CODE REVIEW

**Comment by jgoizueta** [comment]
@iriberri please CR this

File: odbc_fdw.c [hunk 1]
```diff
@@ -769,58 +769,32 @@ getQuoteChar(SQLHDBC dbc, StringInfoData *q_char)
...
 	}
+	return sep;
+}
+
+static void odbcConnStr(StringInfoData *conn_str, odbcFdwOptions* options)
+{
+	bool sep = FALSE;
+	initStringInfo(conn_str);
+	if (!is_blank_string(options->dsn))
+	sep = appendConnAttribute(sep, conn_str, "DSN",      options->dsn);
```
Comment (iriberri) [inline odbc_fdw.c]: Check indent here

Comment (jgoizueta) [inline odbc_fdw.c]: Good catch: the code *works* but  that `if` wasn't meant to be there!

**Comment by iriberri** [comment]
:+1:

File: odbc_fdw.c [hunk 2]
```diff
@@ -769,58 +769,32 @@ getQuoteChar(SQLHDBC dbc, StringInfoData *q_char)
 	appendStringInfo(q_char, "%s", (char *) quote_char);
 }
 
-static void odbcConnStr(StringInfoData *conn_str, odbcFdwOptions* options)
+static bool appendConnAttribute(bool sep, StringInfoData *conn_str, const char* name, const char* value)
 {
-	bool sep = FALSE;
 	static char *sep_str = ";";
```
Comment (iriberri) [inline odbc_fdw.c]: static const

**Pull request merged**